modules/pricer/src/main/java/com/opengamma/strata/pricer/impl/option/SabrInterestRateParameters.java

```diff
@@ -215,21 +215,23 @@ public double volatility(double expiry, double tenor, double strike, double forw
   }
 
   /**
-   * Returns the volatility sensitivity to forward, strike and the SABR model parameters.
+   * Calculates the volatility and associated sensitivities.
    * <p>
-   * The derivatives are stored in an array with
-   * [0] Derivative w.r.t the forward,
-   * [1] the derivative w.r.t the strike,
-   * [2] the derivative w.r.t. to alpha,
-   * [3] the derivative w.r.t. to beta,
-   * [4] the derivative w.r.t. to rho, and
-   * [5] the derivative w.r.t. to nu.
+   * The derivatives are stored in an array with:
+   * <ul>
+   * <li>[0] Derivative w.r.t the forward
```

Comment:
"Derivative" on this line and "the derivative" on the others.

Also shouldn't it be "WRT" or "W.R.T"?